src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/SecurityConfiguration.kt

```diff
@@ -30,6 +31,7 @@ import kotlin.streams.toList
 
 @EnableWebSecurity
 @ConditionalOnWebApplication
+@EnableGlobalMethodSecurity(prePostEnabled = true)
```

Comment:
This is required to enforce role based access at the class and/or method level in Spring.